install/kubernetes/cilium/charts/preflight/templates/deployment.yaml

```diff
@@ -69,14 +69,12 @@ spec:
       serviceAccount: cilium-pre-flight
       serviceAccountName: cilium-pre-flight
       terminationGracePeriodSeconds: 1
+{{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | trim | nindent 8 }}
+{{- end }}
+{{- with .Values.tolerations }}
       tolerations:
-        - effect: NoSchedule
-          key: node.kubernetes.io/not-ready
-        - effect: NoSchedule
-          key: node-role.kubernetes.io/master
-        - effect: NoSchedule
-          key: node.cloudprovider.kubernetes.io/uninitialized
-          value: "true"
-        - key: CriticalAddonsOnly
-          operator: "Exists"
```

Comment:
I would prefer to preserve these in the default values.yaml file for preflight.

Reply:
sounds good! I put them back 👍